cpp/velox/memory/VeloxMemoryManager.cc

```diff
@@ -242,6 +242,24 @@ const MemoryUsageStats VeloxMemoryManager::collectMemoryUsageStats() const {
   return stats;
 }
 
+namespace {
+int64_t shrinkVeloxMemoryPool(velox::memory::MemoryPool* pool, int64_t size) {
+  std::string poolName{pool->root()->name() + "/" + pool->name()};
+  std::string logPrefix{"Shrink[" + poolName + "]: "};
+  DLOG(INFO) << logPrefix << "Trying to shrink " << size << " bytes of data...";
+  DLOG(INFO) << logPrefix << "Pool has reserved " << pool->currentBytes() << "/" << pool->root()->reservedBytes() << "/"
+             << pool->root()->capacity() << "/" << pool->root()->maxCapacity() << " bytes.";
+  DLOG(INFO) << logPrefix << "Shrinking...";
```

Comment:
this log is confusing, could we remove it?

Reply:
> DLOG(INFO) << logPrefix << "Shrinking...";

Did you mean this line?

Reply:
yeah, this line has no meaning.